provisioning/client.crt

```diff
@@ -0,0 +1,25 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Directory name is confusing: consider moving it under travis or such

Reply:
Also, do the certificates in here expire, by chance? Make sure they don't

Reply:
in 100 years

Reply:
well it's not just travis it's also for the vagrant machine so i don't think it makes much sense but I can add a readme

Reply:
Something like moving this under tests is also OK

Reply:
ill move it to vagrant :D

Reply:
As discussed on chat, the reason why I'm asking for this to be moved at least under `tests` or such is because the reader of the package will click on this dir and wonder what it is about in first place: let's remove that doubt.